docs/pages/blog/2019-developer-survey-results.md

```diff
@@ -34,7 +34,7 @@ so we know they love us really. 😘
 
 ### 2. Who do you think would most benefit from MUI?
 
-This word cloud was generated with [wordclouds.com](https://wordclouds.com).
+This word cloud was generated with wordclouds.com.
```

Comment:
Can it be solved with `rel="nofollow"`?
```suggestion
This word cloud was generated with <a rel="nofollow" href="https://wordclouds.com">wordclouds.com</a>.
```

Reply:
For context, I didn't do this change for the backlink, but because `curl -I https://wordclouds.com` returns a 301. I figured that by removing the hard link, we would never have to come back to it again if they change their mind about using a naked domain.

Reply:
oh ok, I wasn't aware of any negative SEO implications on our side by linking to a 3rd party through a redirect.

Reply:
I think that the main negative effect is of second-order. In the first order, it makes the navigation a bit slower and consumes Google Bot's time but I'm not sure it's significant. My main objective is to clean the ahrefs reports, to make it easier for us to not miss when there is a real negative SEO implication.